.woodpecker/social.yaml

```diff
@@ -16,10 +16,17 @@ steps:
       ai_token:
         from_secret: openai_token
       ai_prompt: |
-        We want to present the next version of our app on Twitter.
+        We want to present the next version of our app on Mastodon.
```

Comment:
Used Twitter for the prompt to get a shorter and more precise result.

Reply:
I still think it is quite nice to use an llm to summarize as if done correctly it can phrase a quite nice text (most of the time at a native speaker level and without spelling mistakes). Having the option to highlight specific changes will probably enhance the generation and til then I would just use what we currently have.